docs/components/input.html

```diff
@@ -102,42 +103,108 @@ <h3 class="main-subhead">Input fields</h3>
-                        style="width: 100%; height: 420px; border:0; transform: scale(1); overflow:hidden;"
+                        src="https://carbon.now.sh/embed?bg=rgba%2846%2C54%2C62%2C1%29&t=blackboard&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=10px&dsblur=26px&wc=true&wa=true&pv=41px&ph=47px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522input-container%2520input-success%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Clabel%2520for%253D%2522email%2522%2520class%253D%2522input-label%2522%253EEmail%2520Address%2520%253Cspan%2520class%253D%2522required-mark%2522%253E*%253C%252Fspan%253E%253C%252Flabel%253E%250A%2520%2520%2520%2520%2520%2520%253Cinput%2520type%253D%2522email%2522%2520id%253D%2522email%2522%2520class%253D%2522input-field%2522%2520placeholder%253D%2522xyz%2540compactui.com%2522%2520required%253E%250A%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522input-success-msg%2522%253EValid%2520Email%2520Id%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522far%2520fa-check-circle%2520input-success-icon%2522%253E%253C%252Fi%253E%253C%252Fspan%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522input-container%2520input-error%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Clabel%2520for%253D%2522password%2522%2520class%253D%2522input-label%2522%253EPassword%2520%253Cspan%2520class%253D%2522required-mark%2522%253E*%253C%252Fspan%253E%253C%252Flabel%253E%250A%2520%2520%2520%2520%2520%2520%253Cinput%2520type%253D%2522password%2522%2520for%253D%2522password%2522%2520class%253D%2522input-field%2522%2520required%253E%250A%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522input-error-msg%2522%253EIncorrect%2520Password.%2520Please%2520try%2520again%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fa%2520fa-solid%2520fa-exclamation%2520input-error-icon%2522%253E%253C%252Fi%253E%253C%252Fspan%253E%250A%253C%252Fdiv%253E%250A%253Cbutton%2520class%253D%2522btn%2520btn-primary%2522%253ESubmit%253C%252Fbutton%253E"
+                        style="width: 100%; height: 411px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                    </iframe>

```

Comment:
You might want to delete one closing tag.

Reply:
Done